lib/workos/user_management.rb

```diff
@@ -746,6 +573,106 @@ def list_auth_factors(user_id:)
         )
       end
 
+      # Sends a verification email to the provided user.
+      #
+      # @param [String] id The unique ID of the User whose email address will be verified.
+      #
+      # @return WorkOS::UserResponse
+      sig do
+        params(
+          id: String,
+        ).returns(WorkOS::UserResponse)
+      end
+      def send_verification_email(id:)
+        response = execute_request(
+          request: post_request(
+            path: "/user_management/users/#{id}/email_verification/send",
```

Comment:
nit: single quote

Reply:
Double quote is required here to perform string interpolation. By changing to single, then the tests start breaking.